### PR TITLE
Travis badge

### PR DESCRIPTION
**Pull request by rvanheest**
travis badge

@DANS-KNAW/easy for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 easy-datacite
 =============
 
+[![Build Status](https://travis-ci.org/DANS-KNAW/easy-datacite.svg?branch=master)](https://travis-ci.org/DANS-KNAW/easy-datacite)
```
Comment (PaulBoon) [inline README.md]: Should we put this in our easy-module-archetype?

Comment (rvanheest) [inline README.md]: Nope, this only applies to the public projects that have Travis support. And even then the URL is very specific...

Comment (PaulBoon) [inline README.md]: I guess that in an archetype you have ways to insert the project name into a string somehow. 
And non-public projects should be kept to minimum, most likely the exception...

Comment (rvanheest) [inline README.md]: Agree with both. Feel free to do a pull request on the archetype project! Regarding non-public: every project starts off this way during initial development. Then, at some random point in time we make it public...

Comment (PaulBoon) [inline README.md]: ...  maybe you can hear my slow retreating footsteps

Comment (rvanheest) [inline README.md]: No, no, please feel free to contribute! That would be greatly appreciated!

**Pull request merged**